colossalai/auto_parallel/meta_profiler/metainfo.py

```diff
@@ -91,11 +92,17 @@ def compute_metainfo(self):
         Compute meta info based on sharding strategy and the given target function.
         """
 
-        assert meta_register.has(self._target), f'{self._target} not found in the meta registry'
-        meta_func = meta_register.get(self._target)
+        assert meta_register.has(self._target.__class__), f'{self._target.__class__} not found in the meta registry'
+        meta_func = meta_register.get(self._target.__class__)
 
         # construct args for meta_func
         args = [self.compute_sharded_tensor(k, v) for k, v in self._strategy.sharding_specs.items()]
 
+        # construct kwargs
+        if self.target in INPLACE_MODULE:
+            kwargs = {'inplace': self.target.inplace}
+        else:
+            kwargs = {'inplace': False}
```

Comment:
What will happen is it is relu and inplace is False?

Reply:
The ReLU with `inplace=False` will consume an extra space

Reply:
Will meta info differentiate these two cases (inplace True and False)?

Reply:
Sure, `inplace=False` will create a new tensor in forward phase, while `inplace=True` will not